mars_api/models.py

```diff
@@ -95,15 +95,16 @@ class Meta:
             ),
             models.CheckConstraint(
                 check=models.Q(
-                    corporation__in=[corp[0] for corp in dict(CORPORATIONS).items()]
+                    corporation__in=set(dict(CORPORATIONS))
                 ),
                 name="Only defined corporations are allowed.",
             ),
         ]
 
     def __str__(self):
         s = self.player.nickname + "," if self.player else "<no player>"
+        g = self.game
         return f"""player nickname: {s},
-        game_id: {self.game_id},
-        game date: {self.game.date},
-        corporation: {self.corporation}."""
+        game_id: {g.game_id},
+        game date: {g.game.date},
```

Comment:
In terms of queries, this doesn't make a difference.Try testing this out with `django_assert_num_queries` fixture from `pytest-django`?